packages/micropip/src/micropip/_micropip.py

```diff
@@ -371,7 +402,11 @@ def find_wheel(
 del _PackageManager
 
 
-def install(requirements: str | list[str], keep_going: bool = False):
+def install(
+    requirements: str | list[str],
+    keep_going: bool = False,
+    with_credentials: bool = False,
```

Comment:
Maybe still have,
```
credentials : Optional[str] = None
```
and by default not pass them at all to fetch, otherwise pass the value given by the user?

Also please update the docstring accordingly below